packages/shared/src/components/cards/ad/AdList.tsx

```diff
@@ -42,21 +44,19 @@ export const AdList = forwardRef(function AdCard(
     <FeedItemContainer
       domProps={domProps}
       ref={ref}
-      flagProps={{
```

Comment:
I know Tomer mentioned this, but still not sure if it makes sense to remove this for list desktop.
Wondering if he understood this right.

Anyway no blocker we can show them on preview maybe?